contents/docs/alerts/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Alerts
+sidebar: Docs
+showTitle: true
+availability:
+  free: partial
+  selfServe: full
+  enterprise: full
+---
+
+
+Alerts enable you to monitor your insights and get notified when the value of the insight breaches a threshold that you've set (for example, number of unique visitors to your site increased by 5% week over week).
+Currently alerts are supported on all [trends](https://posthog.com/docs/product-analytics/trends) (support for [funnels](https://posthog.com/docs/product-analytics/funnels) coming soon!).
+
+
+## How to create an alert?
```

Comment:
I can use Screen Studio to record a little video here. There's a lot of steps but they're all pretty straightforward and quick.

Reply:
This will be amazing, lot easier for users to see it one go.
Also in case you need inspiration, [this](https://www.youtube.com/watch?v=JwFzKlMP-Mc) is what Mixpanel's video looks like